account_loan_vendor_payment/wizard/account_payment_register.py

```diff
@@ -0,0 +1,40 @@
+from odoo import _, fields, models
+
+
+class AccountPaymentRegister(models.TransientModel):
+    _inherit = "account.payment.register"
+
+    def _get_default_loan_journal_currency_id(self):
+        return self.env.company.currency_id.id
+
+    loan_payment = fields.Boolean("Open Loan")
+    check_bills_currency = fields.Boolean(default=False)
```

Comment:
**nitpick**: maybe we should explicitly specify the default for both of them, for consistency